image/gimp_brush.ksy

```diff
@@ -0,0 +1,38 @@
+meta:
+  id: gimp_brush
+  title: GIMP (GNU Image Manipulation Program) brush version 2 file
+  file-extension: gbr
+  license: CC0-1.0
+  endian: be
+doc-ref: https://gitlab.gnome.org/GNOME/gimp/-/raw/441631322be109da6489b2aad670bdba916315c0/devel-docs/gbr.txt
+seq:
+  - id: len_header
+    type: u4
+  - id: header
+    type: header
+    size: len_header
+types:
+  header:
+    seq:
+      - id: version
+        type: u4
+      - id: width
+        type: u4
+      - id: height
+        type: u4
+      - id: bytes_per_pixel
+        type: u4
```

Comment:
As @KOLANICH implicitly suggests in https://github.com/kaitai-io/kaitai_struct_formats/pull/427#discussion_r586804286, it really makes sense to bind this field with an enum, because only values `1` and `4` can be used (see [`devel-docs/gbr.txt:41`](https://github.com/GNOME/gimp/blob/0ef5d672b7bd81b50e38f3cfb4bdce2df80f5458/devel-docs/gbr.txt#L41)):

```ksy
          cases:
            color_depth::grayscale: pixel_gray
            color_depth::rgba: pixel_rgba
```

So you should create the following enum:

```ksy
enums:
  color_depth:
    1: grayscale
    4: rgba
```

and attach it to `bytes_per_pixel`:

```diff
       - id: bytes_per_pixel
         type: u4
+        enum: color_depth
```

If you need `bytes_per_pixel` as an integer again (e.g. for the calculation of `body_size`), you can use the enum method [`to_i`](https://doc.kaitai.io/user_guide.html#_enums):

```diff
 instances:
   body_size:
-    value: header.width * header.height * header.bytes_per_pixel
+    value: header.width * header.height * header.bytes_per_pixel.to_i
```

Reply:
this has been fixed (properly I hope)